packages/lib-classifier/src/store/FeedbackStore.spec.js

```diff
@@ -92,4 +92,14 @@ describe('Model > FeedbackStore', function () {
       expect(feedback.rules).to.be.empty
     })
   })
+
+  describe('messages', function () {
+    xit('should return an array of feedback messages', function () {
```

Comment:
Can you use 'it' instead of 'xit'? I'd rather that Travis CI fails for an incomplete PR. If Travis passes the changes, it might look like the PR is ready to merge.